.github/workflows/delete-model-runs.yaml

```diff
@@ -0,0 +1,58 @@
+# Workflow that can be manually dispatched to delete test model runs that
+# do not need to be persisted indefinitely.
+#
+# Gated such that it's impossible to delete runs older than the current
+# assessment cycle, where each assessment cycle starts in April.
+
+name: delete-model-runs
+
+on:
+  workflow_dispatch:
+    inputs:
+      run-ids:
+        description: >
+          Comma-delimited list of IDs of model runs to delete (no spaces). Note
+          that the workflow assumes these IDs correspond to model runs for the
+          current upcoming assessment cycle, and if that's not the case the
+          deletion script will raise an error.
+        required: true
+        type: string
+
+jobs:
+  delete-model-runs:
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to interact with GitHub's OIDC Token endpoint so we can auth AWS
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout repo code
+        uses: actions/checkout@v4
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install system dependencies
+        run: sudo apt-get install libgit2-dev
+        shell: bash
+
+      - name: Disable renv sandbox to speed up install time
+        run: echo "RENV_CONFIG_SANDBOX_ENABLED=FALSE" >> .Renviron
+        shell: bash
+
+      - name: Setup renv
+        uses: r-lib/actions/setup-renv@v2
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_MODEL_DELETION_ARN }}
+          aws-region: us-east-1
+
+      - name: Delete model runs
+        run: Rscript ./R/delete_current_year_model_runs.R "$RUN_IDS"
+        shell: bash
+        env:
+          RUN_IDS: ${{ inputs.run-ids }}
```

Comment:
**nitpick:** Can't you just pass the input directly here?

```suggestion
        run: Rscript ./R/delete_current_year_model_runs.R ${{ inputs.run-ids }}
        shell: bash
```

Reply:
This would be indeed be cleaner! I decided to switch the format of the `run-ids` input variable to expect a space-separated string in a4895c20d2b44cad197df855ac7cdbeebbbe9616, however, so I made a slightly different change here, instead using the `RUN_IDS` env var to strip spaces and replace them with commas. (My thinking is that spaces provide a clearer interface for the GitHub workflow, but on the off chance we ever need to run `R/delete_current_year_model_runs.R` manually, a single space-delimited string would be a much more confusing command line interface than a single comma-delimited string. Accepting a space-delimited string in the GitHub UI but a comma-delimited string via the command line seems like a reasonable compromise to me.)